packages/ui-library/src/theme/variants/chip-variants.ts

```diff
@@ -1,13 +1,18 @@
 import type { Components } from '@mui/material/styles';
 import { colors } from '../colors';
+import { SPACING } from '../constants';
 
 export const buildChipVariant = (mode: 'light' | 'dark'): Components => ({
   MuiChip: {
     styleOverrides: {
+      label: {
+        padding: 0,
```

Comment:
we sure this doesnt break other stuff?

Reply:
I have checked all Chip implementations and they look fine! I have also improved the Activity pending status
<img width="318" alt="Screenshot 2024-08-22 at 5 06 35 PM" src="https://github.com/user-attachments/assets/1f125877-a4f1-4df5-b446-d33a5e8c415a">